cranelift/codegen/src/isa/zkasm/inst/mod.rs

```diff
@@ -955,7 +967,10 @@ impl Inst {
                     }
                 }
             }
-
+            &Inst::MulArith { rd, rs1, rs2 } => {
+                // TODO: I am not very understand what should be here
```

Comment:
This is the disassembly of the instruction. Should output the textual representation of the instruction as well.